examples/markdown.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+import sycamore
+from sycamore.transforms.partition import ArynPartitioner
+
+docs = (
+    sycamore.init()
+    .read.binary(sys.argv[1:], binary_format="pdf")
+    .partition(partitioner=ArynPartitioner(extract_table_structure=True,
+                                           use_partitioning_service=False))
+    .markdown()
+    .explode()
+    .take()
+)
+
+print(docs[1].text_representation)  # doc zero isn't real
```

Comment:
I'm confused. I don't understand what the explode does here. If I don't do the explode, is the document text_representation set to the markdown?

Reply:
If you don't explode, the text_representation is in the Element.  I did it this way because I didn't think converting to Markdown should also do an implicit explode().  I think people are used to seeing explode in pipelines.  It would be easy to change, if needed.